src/main/java/org/springframework/data/elasticsearch/client/util/RequestConverters.java

```diff
@@ -891,6 +893,25 @@ public static Request deleteTemplate(DeleteIndexTemplateRequest deleteIndexTempl
 		return request;
 	}
 
+	public static Request getFieldMapping(GetFieldMappingsRequest getFieldMappingsRequest) {
+		String[] indices = getFieldMappingsRequest.indices() == null ? Strings.EMPTY_ARRAY : getFieldMappingsRequest.indices();
+		String[] fields = getFieldMappingsRequest.fields() == null ? Strings.EMPTY_ARRAY : getFieldMappingsRequest.fields();
+
+		final String endpoint = new EndpointBuilder().addCommaSeparatedPathParts(indices)
+				.addPathPartAsIs("_mapping").addPathPartAsIs("field")
+				.addCommaSeparatedPathParts(fields)
+				.build();
+
+		Request request = new Request(HttpMethod.GET.name(), endpoint);
+
+		RequestConverters.Params parameters = new Params(request);
+		parameters.withIndicesOptions(getFieldMappingsRequest.indicesOptions());
+		parameters.withIncludeDefaults(getFieldMappingsRequest.includeDefaults());
+		parameters.withLocal(getFieldMappingsRequest.local());
```

Comment:
`org.elasticsearch.client.indices.GetFieldMappingsRequest#local()` is deprecated since 7.8.0. The [documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-get-field-mapping.html#get-field-mapping-api-query-params) states: 

> Deprecated in 7.8.0. This parameter is a no-op and field mappings are always retrieved locally

So we should omit this parameter here.

Reply:
Fixed.